src/parse_error.ml

```diff
@@ -1,4 +1,18 @@
-let capitalize_ascii = Astring.String.Ascii.capitalize
+(* This is taken from the ocaml stdlib (more or less) - only here because we support
+   4.02 and capitalize_ascii only arrived in 4.03. When we drop support
+   for 4.02 we can remove the following 3 functions *)
+let uppercase_ascii =
+  let open Char in
+  function 'a' .. 'z' as c -> unsafe_chr (code c - 32) | c -> c
```

Comment:
You can also use `String.capitalize` and disable the warning.

Reply:
Hasn't `String.capitalize` gone entirely now?

Reply:
I actually checked in 5.1 before making the comment but it seems I managed to check in 4.14.1 😂. Sorry for the noise.